db/schema.rb

```diff
@@ -0,0 +1,29 @@
+# encoding: UTF-8
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 20160824165957) do
+
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "education_benefits_claims", force: :cascade do |t|
+    t.datetime "submitted_at"
+    t.datetime "processed_at"
+    t.json     "form",         null: false
```

Comment:
This is the form that will have SSN & Routing info. As a dev, I'd rather get this in and have something to develop against while we work on figuring out how to store sensitive info before releasing to production and collecting real data.